tests/integration_tests/test_simple_features.py

```diff
@@ -162,22 +163,24 @@ def test_feature(input_test_feature, output_test_feature,
 @pytest.mark.parametrize(
     'input_test_feature, output_test_feature',
     [
-        ([category_feature()],
-         [binary_feature(), binary_feature()]),
-        ([category_feature()],
-         [category_feature(vocab_size=5), category_feature(vocab_size=7)]),
-        ([category_feature()],
-         [numerical_feature(), numerical_feature()]),
-        ([category_feature()],
-         [sequence_feature(vocab_size=5), sequence_feature(vocab_size=7)]),
-        ([set_feature(vocab_size=5)],
-         [set_feature(vocab_size=5), set_feature(vocab_size=7)]),
-        ([category_feature()],
-         [text_feature(vocab_size=5), text_feature(vocab_size=7)]),
-        ([category_feature()],
-         [vector_feature(), vector_feature()]),
-        ([vector_feature()],
-         [vector_feature(), vector_feature()]),
+        # ([category_feature()],
+        #  [binary_feature(), binary_feature()]),
+        # ([category_feature()],
+        #  [category_feature(vocab_size=5), category_feature(vocab_size=7)]),
+        # ([category_feature()],
+        #  [numerical_feature(), numerical_feature()]),
+        # ([category_feature()],
+        #  [sequence_feature(vocab_size=5), sequence_feature(vocab_size=7)]),
+        # ([set_feature(vocab_size=5)],
+        #  [set_feature(vocab_size=5), set_feature(vocab_size=7)]),
+        # ([category_feature()],
+        #  [text_feature(vocab_size=5), text_feature(vocab_size=7)]),
+        # ([category_feature()],
+        #  [vector_feature(), vector_feature()]),
+        # ([vector_feature()],
+        #  [vector_feature(), vector_feature()]),
+        ([bag_feature()],
+         [vector_feature()]),
```

Comment:
Could you add a list of vector features for the output here (since the integration test is for multiple outputs)?

Reply:
Done.